packages/blockchain-link/src/workers/electrum/client/json-rpc.ts

```diff
@@ -1,5 +1,5 @@
 import { EventEmitter } from 'events';
-import { throwError } from '@trezor/utils';
+import { throwError } from '@trezor/utils/lib';
```

Comment:
@trezor/utils is a public package. this fix and following should not be needed

Reply:
agree, I already found better solution

Reply:
fixed